docs/how-to-guides/block-tutorial/writing-your-first-block-type.md

```diff
@@ -7,7 +7,7 @@ This guide takes you through creating a basic block to display a message in a po
 There are two main types of blocks: static and dynamic, this guide focuses on static blocks. A static block is used to insert HTML content into the post and save it with the post. A dynamic block builds the content on the fly when rendered on the front end, see the [dynamic blocks guide](/docs/how-to-guides/block-tutorial/creating-dynamic-blocks.md).
 
 <div class="callout callout-alert">
-This guide focuses on just the block, see the [Create a Block tutorial](/docs/getting-started/create-block/README.md) for a complete setup.
+This guide focuses on just the block, see the <a href="/docs/getting-started/create-block">Create a Block tutorial</a> for a complete setup.
```

Comment:
```suggestion
This guide focuses on just the block, see the <a href="https://developer.wordpress.org/block-editor/getting-started/create-block/">Create a Block tutorial</a> for a complete setup.
```

If it is a relative path, when this change is reflected in the handbook, the following URL that does not exist will be accessed.

```
https://developer.wordpress.org/docs/getting-started/create-block
```

It's better to use the full URL.